blessings/__init__.py

```diff
@@ -94,8 +94,13 @@ def __init__(self, kind=None, stream=None, force_styling=False):
             # init sequences to the stream if it has a file descriptor, and
             # send them to stdout as a fallback, since they have to go
             # somewhere.
-            setupterm(kind or environ.get('TERM', 'unknown'),
-                      self._init_descriptor)
+            try:
+                setupterm(kind or environ.get('TERM', 'dumb'),
```

Comment:
Does 'dumb' help, over how 'unknown' used to behave? I honestly can't figure out why I put 'unknown' in there in the first place.

Reply:
for me at least, `unknown` isn't a valid terminal in my term-info and causes a crash as well (which is caught in the except, but I'd rather avoid that if possible). While we can't guarantee that dumb will exist either, I think it has a much higher chance of working (as it's defined on pretty much everything).